docs/data/material/getting-started/templates/sign-in/SignIn.tsx

```diff
@@ -39,73 +107,223 @@ export default function SignIn() {
-                  {"Don't have an account? Sign Up"}
-                </Link>
+            </Typography>
+            <Box
+              component="form"
+              onSubmit={handleSubmit}
+              noValidate
+              sx={{
+                display: 'flex',
+                width: '100%',
+                flexDirection: 'column',
+                gap: 2,
+              }}
+            >
+              <Grid container spacing={2}>
+                <Grid item xs={12}>
+                  <TextField
+                    error={emailError}
+                    helperText={emailErrorMessage}
+                    id="email"
+                    type="email"
+                    name="email"
+                    placeholder="Email address"
+                    autoComplete="email"
+                    autoFocus
+                    required
+                    fullWidth
+                    variant="outlined"
+                    color={emailError ? 'error' : 'primary'}
+                  />
+                </Grid>
+                <Grid item xs={12}>
+                  <TextField
+                    error={passwordError}
+                    helperText={passwordErrorMessage}
+                    name="password"
+                    placeholder="Password"
+                    type="password"
+                    id="password"
+                    autoComplete="current-password"
+                    autoFocus
+                    required
+                    fullWidth
+                    variant="outlined"
+                    color={passwordError ? 'error' : 'primary'}
+                  />
+                </Grid>
+                <Grid item xs={12}>
+                  <Link component="button" onClick={handleClickOpen} variant="body2">
+                    Forgot password?
+                  </Link>
+                </Grid>
               </Grid>
-            </Grid>
-          </Box>
-        </Box>
-        <Copyright sx={{ mt: 8, mb: 4 }} />
-      </Container>
+              <ForgotPassword open={open} handleClose={handleClose} />
+              <Button
+                type="submit"
+                fullWidth
+                variant="contained"
+                onClick={validateInputs}
+              >
+                Sign In
+              </Button>
+              <Link
+                href="/material-ui/getting-started/templates/sign-up/"
+                variant="body2"
+                sx={{ alignSelf: 'center' }}
+              >
+                {"Don't have an account? Sign Up"}
+              </Link>
+            </Box>
+            <Divider>Or</Divider>
+            <Box sx={{ display: 'flex', flexDirection: 'column', gap: 2 }}>
+              <Button
+                type="submit"
+                fullWidth
+                variant="outlined"
+                color="secondary"
+                onClick={() => alert('Sign in with Google')}
+                startIcon={
+                  <img
```

Comment:
I believe these seem to be more accurate for this kind of button, following their brand guidelines. If I'm not mistaken, it comes with the sign in API.

Reply:
I'd then create a custom SVG icon using [SvgIcon](https://mui.com/material-ui/icons/#svgicon) instead of importing the SVG through a thrid-party source